File: DemoMethods/Indexes/CategoryDescription.cs
using System.Linq;
using DemoMethods.Entities;
using Raven.Abstractions.Indexing;
using Raven.Client.Indexes;

namespace DemoMethods.Indexes
{
    public class CategoryDescription : AbstractIndexCreationTask<Category>
    {
        public CategoryDescription()
        {
            Map = categories => from category in categories
                                select new
                                {
                                    category.Description
                                };

            Stores.Add(x => x.Description, FieldStorage.Yes);
            Analyzers.Add(x => x.Description, "Lucene.Net.Analysis.Standard.StandardAnalyzer");
        }
    }
}

File: DemoMethods/src/main/java/net/ravendb/demo/indexes/CategoryDescription.java
package net.ravendb.demo.indexes;

import net.ravendb.abstractions.indexing.FieldStorage;
import net.ravendb.client.indexes.AbstractIndexCreationTask;
import net.ravendb.demo.entities.QCategory;

public class CategoryDescription extends AbstractIndexCreationTask {

    public CategoryDescription() {
        QCategory category = QCategory.category;

        map = "from category in docs.Categories " +
                "select new {" +
                "   Description = category.Description" +
                "}";

        store(category.description, FieldStorage.YES);
        analyze(category.description, "Lucene.Net.Analysis.Standard.StandardAnalyzer");
    }
}
